addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -510,6 +510,10 @@
                 display: none; // remove ballon
             }
 
+            .fa {
+                font-size: $o-we-sidebar-font-size;
+            }
```

Comment:
Just curious: why was this not needed before 16.3?

Reply:
This is because of MILK, we increased the global font-size from 13px to 14 [here](https://github.com/odoo/odoo/commit/2b32d1431a33dea9052be3be4f9668a92cd0422e#diff-4126caf7f145809a1a80cbb65c7e8f953ccb0969d03158e3dce898dc78c9d46dR11)

The 13px on fa icons was small enough to not create the overflow but was already inconsistent with the other svg height in 16.0